ui/src/app/components/admin/admin-user-component.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+
+import { adminGetProfile } from 'app/pages/admin/admin-egress-audit';
+
+import { AdminUserLink } from './admin-user-link';
+
+export const AdminUserComponent = ({ username, userWithDomain }) => {
```

Comment:
seems redundant to require both of these.  I'd prefer to choose one and convert as needed.  We should really have utility functions for this...